README.md

```
@@ -836,7 +836,7 @@ The available options for watermark tool, the watermark is using the options of
 
 | Property                        | Type                                                                                      | Default (possible values)              | Description                                                                                                                                                                                                                                                                                                                                    |
 | ------------------------------- | ----------------------------------------------------------------------------------------- | -------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| **`gallery`**                   | (string                                                                                   | { url: string, previewUrl: string })[] | []                                                                                                                                                                                                                                                                                                                                             | Watermark images urls which are considered to show a list of available watermarks to be used by the user directly from watermark tab |
+| **`gallery`**                   | string                                                                                    | { url: string, previewUrl: string })[] | []                                                                                                                                                                                                                                                                                                                                             | Watermark images urls which are considered to show a list of available watermarks to be used by the user directly from watermark tab |
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 4; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)